### PR TITLE
Add package.json and update installation directions

### PR DESCRIPTION
**Pull request by spinfal**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```
Comment (Akkjon) [inline .gitignore:2]: I would prefer the package-lock.json to be removed from the .gitignore as it reduces errors by installing the exact same version of dependencies everytime and sometimes makes debugging easier.

File: Readme.md [hunk 2]
```diff
@@ -17,8 +17,17 @@ When hovered over a specific point, the playerlist at that specific time is show
...
+```
+```bash
+npm i
+```
+```bash
+npm start
+```
+or to start with [nodemon](https://www.npmjs.com/package/nodemon):
+```bash
+npm start dev
```
Comment (Akkjon) [inline Readme.md]: it is `npm run dev`

File: .gitignore (hunk 1 above)
Comment (Akkjon) [inline .gitignore]: I would prefer the package-lock.json to be removed from the .gitignore as it reduces errors by installing the exact same version of dependencies everytime and sometimes makes debugging easier.

**Review by Akkjon — CHANGES_REQUESTED** [review]

**Review by spinfal — COMMENTED** [review]
Removed package-lock.json from .gitignore & changed "npm start dev" to "npm run dev"

**Review by Akkjon — APPROVED** [review]

**Pull request merged**